include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.h

```diff
@@ -132,6 +140,39 @@ class TimeIntBDF : public TimeIntBDFBase<Number>
   std::shared_ptr<HelpersALE<Number> const> helpers_ale;
 
 private:
+  template<typename Lambda>
+  void
+  get_quantity_and_times(std::vector<VectorType const *> & quantities,
```

Comment:
should also be `quantities` in the name since we have a  vector of DoF vectors.

Reply:
[1f39cef](https://github.com/exadg/exadg/pull/458/commits/1f39cef8d688232b73b239620a45f409a169e862)